todoist/scenes.py

```diff
@@ -130,6 +189,26 @@ def handle_local_intents(self, request: Request):
         pass
 
 
+class CreateTask(TodoistScene):
+    def reply(self, request):
+        task_content = request.intents[intents.CREATE_TASK]['slots']['what']['value']
+        task_content = task_content[0].upper() + task_content[1:]
+
+        task_due_date = Time.from_request_slot(request, intents.CREATE_TASK, 'when')
+        task = api.add_task(task_content, due_date=task_due_date.isoformat())
+
+        due_date = f"на {Time.make_relative(task_due_date)}" if task_due_date else ""
+        service_text = " ".join(["Создала задачу", due_date])
+        texts = [f"{service_text}:", f'"{task.content}"']
+
+        text = " ".join(texts)
+
+        return self.make_response(text)
+
+    def handle_local_intents(self, request: Request):
```

Comment:
why CreateTask is child of TodoistScene? looks more like TodoistScene is some kind of controller class